usaspending_api/api_docs/api_documentation/advanced_award_search/spending_by_award.md

```diff
@@ -66,7 +66,30 @@ order (**OPTIONAL**): Optional parameter indicating what direction results shoul
 }
 ```
 ### Fields
-The possible fields returned are split by contracts or assistace awards (loans, grants, etc.)
+The possible fields returned are split by contracts or assistance awards (loans, grants, etc.)
+
+#### Possible Award Fields (available for every type of award)
+```
+    'Recipient Name': 'recipient_name',
```

Comment:
I don't see `sai_number` here. It should be added as part of the possible Award Fields for the assistance awards since it doesn't apply to contracts.

Reply:
Same with `cfda_number`

Reply:
added